ftest.sh

```diff
@@ -194,7 +197,7 @@ args+=" $*"
 
 # shellcheck disable=SC2029
 # shellcheck disable=SC2086
-if ! ssh $SSH_KEY_ARGS "${REMOTE_ACCT:-jenkins}"@"${nodes[0]}" "set -ex
+if ! ssh $SSH_KEY_ARGS ${REMOTE_ACCT:-jenkins}@"${nodes[0]}" "set -ex
```

Comment:
Shouldn't the REMOTE_ACCT still be quoted as well as SSH_KEY_ARGS (and line #199 can be removed)
```suggestion
if ! ssh "$SSH_KEY_ARGS" "${REMOTE_ACCT:-jenkins}@${nodes[0]}" "set -ex
```

Reply:
We don't want an empty `$SSH_KEY_ARGS` being an _empty-but-present_ argument to `ssh`. If it's empty/unset the shell should just eat it.

`${REMOTE_ACCT:-jenkins}` doesn't need to be quoted because it provides a value if `$REMOTE_ACCT` is empty/unset.